services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/Permissions.kt

```diff
@@ -27,6 +27,8 @@ enum class PermissionCode(val code: Long) {
 interface PermissionRepository : JpaRepository<PermissionEntity, Int> {
     fun findOneByHankeIdAndUserId(hankeId: Int, userId: String): PermissionEntity?
 
+    fun findAllByHankeId(hankeId: Int): List<PermissionEntity>
```

Comment:
Do we need an index for hankeid fk?

Reply:
It might be useful. Though, I think having an index for the `findOneByHankeIdAndUserId` is more important, since that query is done on every request.

I added indices for both of these.